src/main/java/de/themoep/snap/forwarding/listener/ServerDisconnectListener.java

```diff
@@ -33,6 +33,9 @@ public ServerDisconnectListener(Snap snap) {
 
     @Subscribe(order = PostOrder.LAST)
     public void on(KickedFromServerEvent event) {
+        if (!event.getResult().isAllowed()) {
+            return;
+        }
```

Comment:
Basically the same issue here as on the login event: If some other Velocity plugin went after Snap and allowed the event again then a Bungee plugin running under Snap would never see the event.

Reply:
You're right about this: I think we need to call the BungeeCord event regardless of whether the Velocity one is cancelled.